src/containers/shared/components/Transaction/OracleDelete/TableDetail.tsx

```diff
@@ -0,0 +1,15 @@
+import { useTranslation } from 'react-i18next'
+import { TransactionTableDetailProps } from '../types'
+import { OracleDelete } from './types'
+
+export const TableDetail = ({
+  instructions: tx,
+}: TransactionTableDetailProps<OracleDelete>) => {
+  const { t } = useTranslation()
+  return tx.OracleDocumentID !== undefined ? (
+    <div className="oracle-document-id">
+      <span className="label">{t('oracle_document_id')}: </span>
+      <span className="case-sensitive">{tx.OracleDocumentID}</span>
+    </div>
+  ) : null
```

Comment:
```suggestion
  return (
    <div className="oracle-document-id">
      <span className="label">{t('oracle_document_id')}: </span>
      <span className="case-sensitive">{tx.OracleDocumentID}</span>
    </div>
  )
```
`OracleDocumentID` is a required field.